test/integration/cases/upgrade-with-iam/run.sh

```diff
@@ -61,6 +61,9 @@ validate-file /etc/kubernetes/pki/ca.crt 644 expected-ca-crt
 # Order of items in this file is random, skip checking content of /etc/eks/kubelet/environment
 validate-file /etc/eks/kubelet/environment 644
 
+# Since we are upgrading kubernetes version primarily also check if the checksums of artifacts changed
+generate::checksum-file /usr/bin/kubelet
+generate::checksum-file /usr/local/bin/kubectl
```

Comment:
how can we introduce some assertions that help us verify that this not doing anymore what we intended to fix with this work?

Like for example, verify not deleting stuff from the cni folder (which i think we already did before but hacky) or verify that we don't removing packages like ssm/containerd...